examples/mxnet_mnist.py

```diff
@@ -0,0 +1,121 @@
+# Step 1: import required packages
+import os
+import argparse
+import logging
+import mxnet as mx
+import horovod.mxnet as hvd
+from mxnet.test_utils import get_mnist_ubyte
+
+hvd.init()
+
+# CLI
+parser = argparse.ArgumentParser(description='MXNet MNIST Example')
+parser.add_argument('--batch-size', type=int, default=64,
+                    help='training batch size (default: 64)')
+parser.add_argument('--dtype', type=str, default='float32',
+                    help='training data type (default: float32)')
+parser.add_argument('--gpus', type=str, default='0',
+                    help='number of gpus to use (default: 0)')
+parser.add_argument('--epochs', type=int, default=10,
+                    help='number of training epochs (default: 10)')
+parser.add_argument('--lr', type=float, default=0.001,
+                    help='learning rate (default: 0.001)')
+parser.add_argument('--momentum', type=float, default=0.5,
+                    help='SGD momentum (default: 0.5)')
+args = parser.parse_args()
+
+logging.basicConfig(level=logging.INFO)
+logging.info(args)
+
+context = mx.cpu() if args.gpus is None or args.gpus == '0' \
```

Comment:
Add comment: `# Horovod: pin GPU to local rank.`

Reply:
Done